### ISSUE TITLE
Research: Pokemon Showdown API

### ISSUE THREAD

**Issue by prabhathc** [acxz/PokAI#1]
refer to title

**Comment by acxz** [comment]
Yeah I agree, we need to first establish a way to communicate with Pokemon Showdown to perform simulations.

**Comment by prabhathc** [comment]
connection protocol:
https://github.com/Zarel/Pokemon-Showdown/blob/master/PROTOCOL.md

battle protocol:
https://github.com/Zarel/Pokemon-Showdown/blob/master/sim/SIM-PROTOCOL.md

**Comment by acxz** [comment]
Thoughts: showdown's protocol seems kinda complicated, I think letting poke-env handle the protocol parsing into something more meaningful makes sense.
